website/docs/r/app_group_assignment.html.markdown

```diff
@@ -32,6 +24,17 @@ resource "okta_app_group_assignment" "example" {
 }
 JSON
 }
+
+```
+__NOTE__: When using this resource in conjunction with other application resources (e.g. `okta_app_oauth`) it is advisable to add the following `lifecycle` argument to the associated `app_*` resources to prevent the groups being unassigned on subsequent runs:
```

Comment:
Thanks, @ssttgg for the PR! 

I agree with PR. However, could you re-format the markdown like the example below? Just following HashiCorp markdown [callout standards](https://www.terraform.io/docs/registry/providers/docs.html#calloutsl). 😄 

```markdown
!> **NOTE** When using this resource in conjunction with other application resources (e.g. `okta_app_oauth`) it is advisable to add the following `lifecycle` argument to the associated `app_*` resources to prevent the groups being unassigned on subsequent runs:
```

Reply:
Good call, done!